### ISSUE TITLE
Accessing a route differently than the initialRoute is not working as expected on 3.0.0

### ISSUE THREAD

**Issue by jlubeck** [jonataslaw/getx#294]
**Describe the bug**
So using previous version of Get, I was able to get to 
https://site.com/#/organization-intake without a problem even though the initialRoute is set to 
https://site.com/#/login

I was also using `onGenerateRoute` before

With the newest version of GetX, I had to migrate to `getPages` and now no matter what route I try to open, it always goes to the `initialRoute`

After having the page open, if I THEN go to another route, it works. But it doesn't work if it's the first url that you put in the browser. 

this is how I'm setting up my `GetMaterialApp`:

```dart
GetMaterialApp(
      initialRoute: '/login',
      getPages: [
        GetPage(
          name: '/login',
          page: () => LoginScreen(),
        ),
        GetPage(
          name: '/organization-intake',
          page: () => OrganizationScreen(),
        ),
      ],
    )
```

Is there something else I'm missing in the configuration to make this work as before?

Thank you

**Comment by jonataslaw** [comment]
Fix on 3.1.0

**Comment by jlubeck** [comment]
Looks like it, thanks!
